src/net/yapbam/gui/MainMenuBar.java

```diff
@@ -366,18 +384,38 @@ public void actionPerformed(ActionEvent event) {
 			ExportDialog exportDialog = new ExportDialog(this.frame.getJFrame(), this.frame.getFilteredData());
 			exportDialog.setVisible(true);
 			Exporter exporter = exportDialog.getResult();
-			if (exporter!=null) {
-				JFileChooser chooser = new FileChooser(null);
+			if (exporter != null) {

```

Comment:
It seems that changes opens the dialog in the application's directory instead of the user's document directory.
I prefer to keep the original functionality.